lib/node_modules/@stdlib/_tools/pkgs/deps/lib/sync.js

```diff
@@ -58,7 +58,7 @@ var debug = logger( 'pkg-deps:sync' );
 * var pkgs = [ '/foo/bar/baz' ];
 *
 * var deps = pkgDeps( pkgs );
-* // returns [{...}]
+* // throws <Error>
```

Comment:
```suggestion
* // e.g., returns [{...}]
```

The error here is not intended. This example is intended to be illustrative; hence, we can use `e.g.,` to disable doctesting.